src/lazyload.js

```diff
@@ -0,0 +1,21 @@
+import ArticlesFetcher from "./newssourceComponent.js";
+import SourceFetch from "./sourceFetch.js";
+import urlConstructor from "./config.json";
+import errorPopupHandler from "./errorHandler.js";
+
+export default class lazyload {
```

Comment:
I have doubts that we still need this class - there is no requirement sin the home task for it

Reply:
Does that mean Lazy loading not required for this home task?